osgar/drivers/kloubak.py

```diff
@@ -22,7 +22,8 @@
 AD_CENTER = 419.7 # K2, can be modified by config
 AD_CALIBRATION_DEG = 45  # K2, can be modified by config
 AD_RANGE = -182.5  # K2, can be modified by config
-MAX_JOIN_ANGLE = 68 * math.pi/180 # K2, can be modified by config
+MAX_JOIN_ANGLE_DEG = 68
+MAX_JOIN_ANGLE = MAX_JOIN_ANGLE_DEG * math.pi/180 # K2, can be modified by config
```

Comment:
there is [math.radians(degrees)](https://docs.python.org/3/library/math.html#math.radians) for this kind of conversion

Reply:
Ok, why not.